tensorflow/stream_executor/rocm/rocsolver_wrapper.h

```diff
@@ -88,7 +88,7 @@ namespace wrap {
   __macro(rocsolver_dgetri_batched)         \
   __macro(rocsolver_sgetri_batched)         \
   __macro(rocsolver_zgetri_batched)         \
-  __macro(rocsolver_cpotrf)         	    \
+  __macro(rocsolver_cpotrf)         	      \
```

Comment:
Please remove this change as this extraneous space character is not needed.

Reply:
Hmm...it's strange. It looks normal on my vscode. I can see the extra space in my vim although.